### ISSUE TITLE
[feature:] Increase accessibility of vignettes

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [NOAA-FIMS/FIMS#1051]
### Describe the situation that led to the request and a solution

I was rendering the vignettes locally with `pkgdown::build_articles()` and I received the following messages that indicates we need to accomplish these tasks
- [x] use an aria-label for the fa-brands icon
- [x] ensure all figures have alternative text
- [x] fix the missing logo.png image
- [x] Change the "title" entry in pkgdown\_pkgdown.yml on line 27 to be "FIMS" instead of "Fisheries Integrated Modeling System (FIMS)" because the table of contents is cluttered with the full name

```
✖ Icon "fa-brands fa-github fa-lg" lacks an `aria-label`.
ℹ Specify `aria-label` to make the icon accessible to screen readers.
ℹ Learn more in `vignette(pkgdown::accessibility)`.
This message is displayed once every 8 hours.
Writing `articles/index.html`
Reading vignettes/fims-demo.Rmd
Writing `articles/fims-demo.html`
Missing images in vignettes/fims-demo.Rmd: ../logo.png
ℹ pkgdown can only use images in man/figures and vignettes
✖ Missing alt-text in vignettes/fims-demo.Rmd
• https://noaa-fims.github.io/images/FIMS_logo3.png
• fims-demo_files/figure-html/fit-plots-1.png
• fims-demo_files/figure-html/fit-plots-2.png
• fims-demo_files/figure-html/fit-plots-3.png
• fims-demo_files/figure-html/fit-plots-4.png
• fims-demo_files/figure-html/length-1.png
ℹ Learn more in `vignette(pkgdown::accessibility)`.
Reading vignettes/fims-logging.Rmd
Writing `articles/fims-logging.html`
Missing images in vignettes/fims-logging.Rmd: ../logo.png
ℹ pkgdown can only use images in man/figures and vignettes
✖ Missing alt-text in vignettes/fims-logging.Rmd
• https://noaa-fims.github.io/images/FIMS_logo3.png
• figures/selectivity_logging_entry.png
ℹ Learn more in `vignette(pkgdown::accessibility)`.
Reading vignettes/fims-path-maturity.Rmd
Writing `articles/fims-path-maturity.html`
Missing images in vignettes/fims-path-maturity.Rmd: ../logo.png
ℹ pkgdown can only use images in man/figures and vignettes
✖ Missing alt-text in vignettes/fims-path-maturity.Rmd
• https://noaa-fims.github.io/images/FIMS_logo3.png
• figures/fims-path-maturity-8.png
• figures/fims-path-maturity-1.png
• figures/fims-path-maturity-2.png
• figures/fims-path-maturity-3.png
• figures/fims-path-maturity-4.png
• figures/fims-path-maturity-5.png
• figures/fims-path-maturity-6.png
• figures/fims-path-maturity-7.png
• figures/fims-path-maturity-8.png
ℹ Learn more in `vignette(pkgdown::accessibility)`.
```

The [referenced vignette from {pkgdown}](https://pkgdown.r-lib.org/articles/accessibility.html) gives the following solution for the fa-brand icon
```
cran:
  icon: fab fa-r-project
  href: https://cloud.r-project.org/package=pkgdown
  aria-label: View on CRAN
```
